gravatar/build.gradle.kts

```diff
@@ -75,7 +75,7 @@ android {
 }
 
 dependencies {
-    implementation("com.squareup.okhttp3:okhttp:4.12.0")
+    api("com.squareup.okhttp3:okhttp:4.12.0")
```

Comment:
I have another PR for this: https://github.com/Automattic/Gravatar-SDK-Android/pull/128

Reply:
Merged